common/djangoapps/entitlements/models.py

```diff
@@ -307,26 +311,33 @@ def get_active_entitlements_for_user(cls, user):
         ).select_related('user').select_related('enrollment_course_run')
 
     @classmethod
-    def does_user_have_active_entitlements(cls, user):
+    def get_fullfillable_entitlements(cls, user):
```

Comment:
Update comment for this method to reflect new functionality and parameters taken